.github/workflows/ci_mac_gtk.yml

```diff
@@ -0,0 +1,190 @@
+name: Mac with Gtk3 builds
+
+on:
+  # Allows you to run this workflow manually from the Actions tab, right now no other triggers until proven
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  
+jobs:
+  build-and-test:
+    defaults:
+      run:
+        shell: /usr/bin/arch -arch ${{ matrix.arch }} /bin/bash --noprofile --norc -eo pipefail {0}
+
+    runs-on: ${{ matrix.runner }}
+
+    name: ${{ matrix.name }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: wxGTK on macOS
+          runner: macos-latest
+          arch: arm64
+          configure_flags: --disable-uiactionsim --with-cxx=11 --without-liblzma --enable-debug --disable-sys-libs --with-gtk=3 --with-opengl=no --enable-detect_sm=no
```

Comment:
Just out of curiosity, is OpenGL broken in this build?

Also, not sure why we need to disable LZMA, it will just be auto-disabled if support is not found?

Reply:
the LZMA was a left over from an earlier configure line of mine, I'll remove that. With OpenGL I ran into a build problem, apparently the homebrew version of gtk+ uses Quartz and not X11 anymore.